pages/updates.js

```diff
@@ -47,7 +57,7 @@ class UpdatesPage extends React.Component {
 
     return (
       <div className="UpdatesPage">
-        <Header collective={collective} LoggedInUser={LoggedInUser} />
+        <Header collective={collective} LoggedInUser={LoggedInUser} canonicalURL={getCanonicalURL(collective)} />
```

Comment:
Here for example: `canonicalURL` should look like `/parentSlug/collectiveSlug/updates`, not `/parentSlug/collectiveSlug`